includes/donors/class-give-donor-wall.php

```diff
@@ -327,10 +328,17 @@ private function get_donation_data( $atts = array() ) {
 		// Backward compatibility
 		$donation_id_col = Give()->payment_meta->get_meta_type() . '_id';
 
+		// order by query based on parameter.
+		if ( 'donation_amount' === $query_params['orderby'] ) {
+			$order_by_query = "ORDER BY FIELD( p1.ID, {$donation_ids} )";
```

Comment:
@kakshak can you explain how this query will sort donations on basis of donation amount.